src/gui/app/directives/conditional-effect/conditionDisplay.js

```diff
@@ -73,19 +73,19 @@
 
                 $ctrl.$onInit = function() {
                     getRightSideValueDisplay().then((value) => {
-                        $ctrl.rightSideValueDisplay = value || "[Not Set]";
+                        $ctrl.rightSideValueDisplay = value !== null && value !== undefined ? value : "[Not Set]";
```

Comment:
This works just fine. But note that a single `value != null` check would be sufficient as it only checks null and undefined, it's not a full falsey check (that would catch 0).